### DISCUSSION TITLE
Options for markdown triggers

### DISCUSSION THREAD

**Discussion by iliatapazukk** [remarkjs/remark#896] [Q&A] [answered]
Hello! 
Is there an option to produce bold text only with `** ... **`, while excluding `__ ... __` in react-markdown?

**Comment by wooorm** [comment, marked as answer]
Nope, react-markdown and remark follow markdown, specifically CommonMark (optionally GFM), which don‘t describe such behavior.
You can always try and integrate with the parser (micromark), but it’s going to get complex and I wouldn’t recommend it
